templates/cli/lib/commands/generic.js.twig

```diff
@@ -49,61 +119,74 @@ const whoami = new Command("whoami")
 
 const login = new Command("login")
     .description(commandDescriptions['login'])
+    .option(`-sa, --self-hosted`, `Flag for enabling custom endpoint for self hosted instances`)
     .configureHelp({
         helpWidth: process.stdout.columns || 80
     })
+    .action(actionRunner(loginCommand));
+
+login
+    .command('list')
+    .description("List available logged accounts.")
     .action(actionRunner(async () => {
-        const answers = await inquirer.prompt(questionsLogin)
+        const logins = globalConfig.getLogins();
+        const current = globalConfig.getCurrentLogin();
 
-        let client = await sdkForConsole(false);
+        const data = [...logins.map((login => {
+            return {
+                'Current': login.id === current ? '*' : '',
+                'ID': login.id,
+                'Endpoint': login.endpoint,
+                'Email': login.email
+            };
+        }))];
 
-        await accountCreateEmailPasswordSession({
-            email: answers.email,
-            password: answers.password,
-            parseOutput: false,
-            sdk: client
-        })
+        drawTable(data);
 
-        client.setCookie(globalConfig.getCookie());
+    }));
 
-        let account;
+login
+    .command('change')
+    .description("Change the current account")
+    .option(`-a, --accountId <accountId>`, `Login ID`)
+    .action(actionRunner(async ({ accountId }) => {
+        const loginIds = globalConfig.getLoginIds();
 
-        try {
-            account = await accountGet({
-                sdk: client,
-                parseOutput: false
-            });
-        } catch (error) {
-            if (error.response === 'user_more_factors_required') {
-                const { factor } = await inquirer.prompt(questionsListFactors);
-
-                const challenge = await accountCreateMfaChallenge({
-                    factor,
-                    parseOutput: false,
-                    sdk: client
-                });
-
-                const { otp } = await inquirer.prompt(questionsMfaChallenge);
-
-                await accountUpdateMfaChallenge({
-                    challengeId: challenge.$id,
-                    otp,
-                    parseOutput: false,
-                    sdk: client
-                });
-
-                account = await accountGet({
-                    sdk: client,
-                    parseOutput: false
-                });
-            } else {
-                throw error;
-            }
+        if (!loginIds.includes(accountId)) {
+            throw Error('Login ID not found');
         }
 
-        success("Signed in as user with ID: " + account.$id);
+        globalConfig.setCurrentLogin(accountId);
+        success(`Current account is ${accountId}`);
     }));
 
+login
+    .command('migrate')
```

Comment:
Hmm I dont like migrate command in general. Can we make it automatic? Or have backwards compatibility that combines both keys/files?

Reply:
Hooking migration 
![image](https://github.com/appwrite/sdk-generator/assets/316103/e58f3245-d91f-46c6-8cd0-9cbb2a57ddda)